### PR TITLE
changed store api's --sync-block-duration to 15m

### PR DESCRIPTION
**Pull request by Player256**
* [x] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes
Changed the --sync-block-duration in store api from 3m to 15m as suggested from the discussion #6288

### CODE REVIEW

**Comment by douglascamata** [comment]
@Player256 can you add a changelog entry, please? 🙏

**Comment by Player256** [comment]
> @Player256 can you add a changelog entry, please? 🙏

Yes, sure I will add it.

**Review by douglascamata — APPROVED** [review]
LGTM, thanks! :shipit:

**Review by saswatamcode — DISMISSED** [review]
LGTM! Could you just resolve the conflicts @Player256?

**Review by saswatamcode — DISMISSED** [review]
LGTM!

**Comment by saswatamcode** [comment]
Not quite sure why the GH actions are stuck 🤔

**Comment by Player256** [comment]
should I open another pr?

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#6519](https://github.com/thanos-io/thanos/pull/6519) Reloader: Use timeout for initial apply.
 - [#6509](https://github.com/thanos-io/thanos/pull/6509) Store Gateway: Remove `memWriter` from `fileWriter` to reduce memory usage when sync index headers.
 - [#6556](https://github.com/thanos-io/thanos/pull/6556) Thanos compact: respect block-files-concurrency setting when downsampling
+- [#6539](https://github.com/thanos-io/thanos/pull/6539) Store : Changed `--sync-block-duration` default 3m to 15m.
```
Comment (matej-g) [inline CHANGELOG.md:55]: Is this potentially a breaking change? Should we add a warning / breaking change note in the changelog? cc @douglascamata @saswatamcode

Comment (douglascamata) [reply CHANGELOG.md:55]: Definitely worth the warning of the different default value for those not already changing it. I wouldn't consider it a breaking change though, as changing this default value shouldn't truly break things.

Comment (Player256) [reply CHANGELOG.md:70]: sure 👍

**Comment by fpetkovski** [comment]
Let's merge this, I will follow up with a PR to fix docs.

**Pull request merged**